checkpoint/frontend/README.md

```diff
@@ -39,10 +39,25 @@ at http://localhost:3000.
 
 ## Local Development
 
-### Mock API
-### Local API
-### Staging API
+### MLflow
```

Comment:
@KateDK Can you follow these steps and make sure they work for you?

Reply:
Works! 😁